execinquery.go

```diff
@@ -29,7 +29,7 @@ type linter struct {
 
 func newLinter() *linter {
 	return &linter{
-		commentExp:          regexp.MustCompile("-- [^\n]+\n"),
+		commentExp:          regexp.MustCompile(`-[^\n]*\n`),
```

Comment:
```suggestion
		commentExp:          regexp.MustCompile(`--[^\n]*\n`),
```
I think you accidentally deleted the second dash. :laughing: